flytekit-testing/src/main/java/org/flyte/flytekit/testing/SdkTestingExecutor.java

```diff
@@ -116,6 +116,42 @@ public static SdkTestingExecutor of(
         .build();
   }
 
+  /**
+   * Creates a new {@link SdkTestingExecutor} for testing a {@link SdkDynamicWorkflowTask}. This
+   * method wraps the given dynamic workflow task and its input into a delegating workflow, allowing
+   * the task to be executed and tested in isolation.
+   *
+   * @param task the dynamic workflow task to test
+   * @param input the input to the dynamic workflow task
+   * @param outputType the expected output type of the dynamic workflow task
+   * @param <InputT> the type of the input
+   * @param <OutputT> the type of the output
+   * @return a new {@link SdkTestingExecutor} instance
+   *
+   *     <p>Example usage:</p>
+   *     <pre>{@code
```

Comment:
@honnix it seems like there's a conflict between checkstyle's `JavadocParagraph` module and spotless. When I run `mvn verify` locally, spotless reformat this snippet to be something like this:
```suggestion
   * @return a new {@link SdkTestingExecutor} instance
   *     <p>Example usage:
   *     <pre>{@code
```

But the lack of that empty line before the `<p>` tag makes checkstyle very unhappy. Should we remove that module from the checkstyle config?

Reply:
I will propose a PR to remove checkstyle completely. It has always been creating more troubles than being truly helpful.

Reply:
We removed checkstyle in #320 . Could you please remove the last commit and rebase again?

Reply:
I did that for you.